<ui version="4.0" >
 <class>SnapShot</class>
 <widget class="QWidget" name="SnapShot" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SnapShot</string>
  </property>
  <widget class="QPushButton" name="_snapShotButton" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Snap Shot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>92</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Number of Samples</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="_numSamples" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>70</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum" >
    <number>1</number>
   </property>
   <property name="value" >
    <number>4</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="_frameScale" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value" >
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Frame Scale</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
